frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -9,6 +9,10 @@ Cypress.Cookies.defaults({
   whitelist: ['openshift-session-token', 'csrf-token'],
 });
 
+Cypress.Commands.overwrite('visit', (originalFn, url, options) => {
+  return originalFn(url, { retryOnStatusCodeFailure: true, ...options });
```

Comment:
I'm debating if we should do this or not. I wouldn't expect to have a status code failure, although I guess a problem with ingress could cause it. We might want to dig into the failures more closely before making this change.

Reply:
I'll look for network failures, but what is the harm in retrying on failure?

Reply:
> I'll look for network failures, but what is the harm in retrying on failure?

It masks real bugs. If we have a backend or operator PR that sometimes causes console to respond with an error, we won't catch it.

Reply:
I would leave this out for now until we understand the problem better.

Reply:
Ok, it is removed.